Converters/Tekla/Speckle.Converters.TeklaShared/ToSpeckle/Helpers/PropertiesExtractor.cs

```diff
@@ -19,6 +24,14 @@ public PropertiesExtractor(ReportPropertyExtractor reportPropertyExtractor)
       properties.Add("Report", report);
     }
 
+    Dictionary<string, object?> userDefinedAttributes = _userDefinedAttributesExtractor.GetUserDefinedAttributes(
+      modelObject
+    );
+    if (userDefinedAttributes.Count > 0)
+    {
+      properties.Add("UserDefinedAttributes", userDefinedAttributes);
```

Comment:
minor change, but we've been using an all caps convention with spaces like "User Defined Attributes"